crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -430,6 +448,12 @@ pub struct AuthorityEpochTables {
 
     /// Records the round numbers for which we have written randomness.
     randomness_rounds_written: DBMap<RandomnessRound, ()>,
+
+    /// Tables for recording DKG state for RandomnessManager.
+    pub(crate) dkg_processed_messages: DBMap<PartyId, dkg::ProcessedMessage<PkG, EncG>>,
+    pub(crate) dkg_used_messages: DBMap<u64, dkg::UsedProcessedMessages<PkG, EncG>>,
+    pub(crate) dkg_confirmations: DBMap<PartyId, dkg::Confirmation<EncG>>,
+    pub(crate) dkg_output: DBMap<u64, dkg::Output<PkG, EncG>>,
```

Comment:
can you add a brief comment explaining what each table is for and how/when it is updated?

Reply:
done